### PR TITLE
Parse and render IWA type

### LINKED ISSUE #12: Handle IWA identifiers (and its draft stages)
We need to handle the IWA draft stages.

```
AWI IWA 36
AWI IWA 39
AWI IWA 41
AWI IWA 42
CD IWA 37
CD IWA 37-1
CD IWA 37-2
CD IWA 37-3
CD IWA 40
CD IWA 42
ISO/IWA 32:2019(en)
ISO/WD IWA 19
IWA 14-1:2013
IWA 19:2017
IWA 2 N3
IWA 24:2016
IWA 26:2017
IWA 31:2020
IWA 32:2019
IWA 34:2021
IWA 34:2021(en)
IWA 35:2020
IWA 37-1:2022
IWA 37-2:2022
IWA 37-3:2022
IWA 40:2022(en)
IWA 42
IWA 8:2009
NP IWA 30-2
NP IWA 32
NP IWA 33
PRF IWA 36
PRF IWA 37-1
PRF IWA 37-3
PRF IWA 40
PRF IWA 42
WD IWA 19
```

We need to parse items like:
```
ISO/IWA 32:2019(en)
ISO/WD IWA 19
```

But the correct forms are:
```
IWA 32:2019(en)
WD IWA 19
```

(Updated list)

### PR DESCRIPTION
**Pull request by mico**
closes #12

### CODE REVIEW

File: lib/pubid/iso/identifier/international_workshop_agreement.rb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    class InternationalWorkshopAgreement < Base
+      def_delegators 'Pubid::Iso::Identifier::InternationalWorkshopAgreement', :type
+
+      TYPED_STAGES = {
+        dis: {
+          abbr: "DIWA",
+          name: "Draft International Workshop Agreement",
+          harmonized_stages: %w[40.00 40.20 40.60 40.92 40.93],
+        },
+        fdis: {
```
Comment (ronaldtse) [inline lib/pubid/iso/identifier/international_workshop_agreement.rb:12]: There is no "Final Draft" stage for IWA. This is described in the Typed Stages Excel file:
Screen Shot 2022-12-14 at 10 27 29 PM

Can you please refer to the file?

**Review by ronaldtse — CHANGES_REQUESTED** [review]

File: lib/pubid/iso/identifier/international_workshop_agreement.rb (hunk 1 above)
Comment (mico) [reply lib/pubid/iso/identifier/international_workshop_agreement.rb:12]: I'll add this change, but excel file version I have looks different:
image

Do you have other updates?

Comment (ronaldtse) [reply lib/pubid/iso/identifier/international_workshop_agreement.rb]: I’ve previously uploaded the latest version to a ticket. I’ll have to find that again.

**Comment by ronaldtse** [comment]
@mico could you please help rebase and merge? Thanks.

**Review by ronaldtse — APPROVED** [review]

File: lib/pubid/iso/identifier/international_workshop_agreement.rb (hunk 1 above)
Comment (ronaldtse) [reply lib/pubid/iso/identifier/international_workshop_agreement.rb]: It is now at #176 .

**Pull request merged**